.gitmodules

```diff
@@ -1,12 +1,12 @@
 [submodule "thirdparty/cub"]
 	path = thirdparty/cub
-	url = https://github.com/NVlabs/cub.git
-	branch = b165e1fb11eeea64ccf95053e40f2424312599cc
+	url = https://github.com/rapidsai/thirdparty-cub.git
+	branch = cudf-cub
 [submodule "thirdparty/moderngpu"]
 	path = thirdparty/moderngpu
-	url = https://github.com/moderngpu/moderngpu.git
-	branch = c1fd31df008d79f727483e795b1ee1ce45b782ca
+	url = https://github.com/rapidsai/thirdparty-moderngpu.git
+	branch = cudf-moderngpu
 [submodule "thirdparty/rmm"]
 	path = thirdparty/rmm
 	url = https://github.com/rapidsai/rmm
-	branch = branch-0.5
+	branch = .
```

Comment:
Why was this changed to `.` instead of `branch-0.5`?

Reply:
Because that is shorthand in git to match the current branch name

Reply:
So we don't have to change it with every release and dev branch

Reply:
👍

Reply:
https://git-scm.com/docs/git-submodule#git-submodule---branch